funnel/templates/proposals.html.jinja2

```diff
@@ -0,0 +1,163 @@
+                    {%- endfor -%}
+                    )
+                  {%- endif %}
+                {%- endif %}
+              </li>
+              <li>{{ proposal.commentset.count }} comments</li>
+              <li class="no-border">{{ proposal.datetime.strftime("%a, %e %b") }}</li>
+              {% if proposal.preview_video %}
+                <li class="no-border"><i data-toggle="tooltip" data-placement="top" title="This proposal has a preview video" aria-label="Preview video" class="material-icons icon-middle">play_arrow</i></li>
+              {% endif %}
+              {% if proposal.slides %}
+                <li class="no-border"><i data-toggle="tooltip" data-placement="top" title="This proposal has slides" aria-label="Outstation Speaker" aria-label="Slides" class="material-icons icon-middle">slideshow</i></li>
+              {% endif %}
+            </ul>
+            <hr class="mui-divider">
+          </td>
+        </tr>
+      {% else %}
+        <tr>
+          <td><em>{% trans %}(No sessions have been submitted){% endtrans %}</em></td>
+        </tr>
+      {% endfor %}
+    </tbody>
+  </table>
+{%- endmacro %}
+
+{%- macro page_btns(project) %}
+  <ul class="mui-list--inline">
+    {%- if project.boxoffice_data %}
+      <li class="cta">
+        <a href="#tickets" class="mui-btn mui-btn--raised mui-btn--primary">{% trans %}Participate{% endtrans %}</a>
+      </li>
+    {%- elif project.buy_tickets_url %}
+      <li class="cta">
+        <a href="{{ project.buy_tickets_url }}" class="mui-btn mui-btn--raised mui-btn--primary" target="_blank">{% trans %}Participate{% endtrans %}</a>
+      </li>
+    {%- endif %}
+    {% if project.state.SUBMISSIONS -%}
+      <li class="mui--hidden-xs">
+        <a class="mui-btn mui-btn--raised mui-btn--accent" href="{{ project.url_for('new_proposal') }}">{% trans %}Propose a session{% endtrans %}</a>
+      </li>
+    {% endif %}
+    {% if project.sessions %}
+      <li class="mui--hidden-xs mui--hidden-sm mui--hidden-md {% if project.state.SUBMISSIONS -%}float-btn{% endif %}">
+        <a href="{{ project.url_for('schedule') }}" class="mui-btn mui-btn--raised">{% trans %}View Schedule{% endtrans %}</a>
+      </li>
+    {% endif %}
+  </ul>
+{%- endmacro %}
+
+{% block headline %}
+  {{ project_header(project, page_btns=page_btns(project), current_page='proposals') }}
+{% endblock %}
+
+{% block contentwrapper %}
+  <div class="grid details-section">
+    <div class="grid__col-xs-12 grid__col-md-5 grid__col-lg-4">
+      {%- if project.current_roles.admin or 'checkin-event' in g.permissions %}
+        {{ admin_panel(project) }}
+      {% endif %}
+    </div>
+
+    <div class="grid__col-xs-12 grid__col-md-7 grid__col-lg-8" id="proposals">
+      <div class="grid__col-xs-12">
+        <h2 class="mui--text-display1 mui--text-left project-section__headline">All proposals</h2>
+        {%- if 'view-status' in current_auth.permissions %}
```

Comment:
This should be `current_auth.permissions.view_status`. Backend code will need to rename `view-status` to `view_status`. We should also find a better name for it. Maybe `view_review_state`